.github/workflows/test.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         id: waitFor200
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          max_timeout: 480
+          max_timeout: 960
```

Comment:
I'm pretty sure that most of these failures are not timeouts sadly 😞

Reply:
I'm not sure either if this will help, but I thought we should at least try?

Reply:
After some experimentation with the action I realised this: https://github.com/RecordReplay/devtools/issues/6730#issuecomment-1125171169. So I'm doubling down on this being a timeout issue.

Reply:
Oh, interesting, so that's saying that the check at `480` (for example) might actually be checking the result of the check at `420`? I'm ok with bumping this, but we should watch our for API rate-limiting errors, because that is what prompted me to bump the timeout so high, and if we have more builds pinging we might end up tripping that again.

Reply:
Wait whaaaaaaat? That's not very intuitive! Well, that's great, thanks for finding this, let's 🤞 !

Reply:
One more thought on this: given what you've discovered, would it make sense to bump the timeout to some slightly odd number. Like would making it 961 be any better? It would be nice if the effect of this setting were as close as possible to the actual number we set it to 😆

Reply:
No, that wouldn't help: the action is always waiting for `check_interval`, so the timeout value is effectively rounded to a multiple of `check_interval`. But I have decreased `check_interval` now, so that should help.

Reply:
OK, just a heads up - I very much expect that we will be triggering the API rate limit with this change, but let's see what happens.

Reply:
So *that* was why it was changed from the default. I'll change it back since this is not really necessary (I just did it to make the tests start a *little* earlier)